lib/smack/SmackRep.cpp

```diff
@@ -201,22 +311,22 @@ const Expr* SmackRep::b2p(const llvm::Value* v) {
 const Expr* SmackRep::i2b(const llvm::Value* v) {
   return Expr::fn(I2B, expr(v));
 }
-const Expr* SmackRep::b2i(const llvm::Value* v) {
-  return Expr::fn(B2I, expr(v));
+const Expr* SmackRep::b2i(const llvm::Value* v, unsigned size) {
```

Comment:
I'm kind of confused. The type of `v` should be `i1` otherwise `isBool(v->getType())` in `cast` method does not make any sense. The IR code `%x = zext i1 %y to i32,` got translated to `x = b2i(y)`, which implies that `x` could be any integer size.